larq_compute_engine/tflite/kernels/bconv2d.cc

```diff
@@ -507,26 +414,6 @@ void SetupQuantization(TfLiteContext* context, TfLiteNode* node,
   }
 }
 
-// Helper to get the type to unpack to
-template <typename SrcScalar>
-struct GetUnpackType {};
-
-template <>
-struct GetUnpackType<float> {
-  using type = float;
-};
-
-template <>
-struct GetUnpackType<TBitpacked> {
-  using type = float;
-};
-
-template <>
-struct GetUnpackType<std::int8_t> {
-  using type = std::int8_t;
-};
-
-template <class SrcScalar>
 void OneTimeSetup(TfLiteContext* context, TfLiteNode* node,
                   TfLiteBConv2DParams* params) {
   if (!params->is_filter_repacked || !params->is_padding_correction_cached) {
```

Comment:
I think we can remove the entire 'repacking' section of the code now, because all packing is now canonical.
EDIT: I see that we use the unpacked type for the zero-padding correction, but I think the zero-padding correction also supports packed filters.